src/app/(auth)/sign-up/page.tsx

```
@@ -0,0 +1,12 @@
+import { Metadata } from "next";
+import SignUpView from "@/modules/auth/ui/views/sign-up-view";
+
+export const metadata: Metadata = {
+  title: "Sign Up",
+};
+
+type Props = {};
+
+export default function SignUpPage({}: Props) {
```

Biome (1.9.4)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)